server/controllers/user.controllers.js

```diff
@@ -337,6 +339,25 @@ const UserController = {
       })
       .catch((error) => next(error));
   },
+
+  resendActivationEmail(req, res, next) {
+    let userId;
```

Comment:
This can be written on a single line 😏 
```suggestion
    const userId = req.user.role === USER_ROLE.ADMIN ? req.params.id : req.user._id;
```